modules/040-node-manager/monitoring/prometheus-rules/node-unmanaged.tpl

```diff
@@ -12,15 +12,14 @@
         plk_create_group_if_not_exists__d8_cluster_has_unmanaged_nodes: "D8ClusterHasUnmanagedNodes,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
         plk_grouped_by__d8_cluster_has_unmanaged_nodes: "D8ClusterHasUnmanagedNodes,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
     {{- if .Values.global.modules.publicDomainTemplate }}
```

Comment:
What for do we need this `if` condition if we don't use private docs links anymore. I think we can left only one kind of alert

Reply:
Fixed